.github/workflows/android-feature.yml

```diff
@@ -0,0 +1,30 @@
+# A workflow for controlling the acions taken by CI whenever we push a commit to an open pull request
+
+name: Android SDK Feature CI
+
+# Triggers the workflow whenever a pull request is opened, reopened or has commits pushed to it
+on:
+  pull_request
+
+jobs:
+  # This workflow contains a single job for running our tests
+  test:
+    name: Run Unit Tests
+    
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
```

Comment:
we might want to specify a version to keep things consistent

Reply:
Fair. I will update these to run against ubuntu-20.04 which supports the version of Java we want to build against.